Server/mods/deathmatch/logic/luadefs/CLuaACLDefs.cpp

```diff
@@ -1049,3 +1051,36 @@ int CLuaACLDefs::OOP_isObjectInACLGroup(lua_State* luaVM)
     lua_pushboolean(luaVM, false);
     return 1;
 }
+
+std::variant<std::vector<CAccessControlListGroup*>, bool> CLuaACLDefs::aclObjectGetGroups(SString strObject)
```

Comment:
change type from `SString` to `std::string`, convert it later to "SString" if needed